src/core/Tools.cpp

```diff
@@ -242,6 +242,7 @@ namespace Tools
     bool checkUrlValid(const QString& urlField)
     {
         if (urlField.isEmpty() || urlField.startsWith("cmd://", Qt::CaseInsensitive)
+            || urlField.startsWith("kdbx://", Qt::CaseInsensitive)
```

Comment:
might as well toss file:// up here as well and then we can delete the exception for file down on line 257

Reply:
Sure. Still, it looks file `file://` is also checked that it actually contains something. Maybe a similar check would be needed for `cmd://` and `kdbx://` as well.

Reply:
Agree